### PR TITLE
[DOCS] Group commands example 

### PR DESCRIPTION
**Pull request by Misha-133**
This PR adds an example with command and sub-command groups to interaction framework inro
It should explain how to use `Group` attribute

# Changes
- added codeblock with an example for interaction framework intro

image

### CODE REVIEW

File: docs/guides/int_framework/samples/intro/groupmodule.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{
+    [SlashCommand("ping", "Get a pong")]
+    public async Task PongSubcommand()
+        => await RespondAsync("Pong!");
+
+    [Group("subcommand-group", "Subcommand group description")]
+    public class SubcommandGroupModule : InteractionModuleBase<SocketInteractionContext>
+    {
+        [SlashCommand("echo", "Echo an input")]
+        public async Task PongSubcommand(string input)
```
Comment (csmir) [inline docs/guides/int_framework/samples/intro/groupmodule.cs]: I suggest naming this 'EchoSubcommand' instead for logic

**Review by csmir — CHANGES_REQUESTED** [review]

File: docs/guides/int_framework/samples/intro/groupmodule.cs (hunk 1 above)
Comment (Misha-133) [reply docs/guides/int_framework/samples/intro/groupmodule.cs]: Oops, didn't notice that i forgot to change the name ._.

File: docs/guides/int_framework/samples/intro/groupmodule.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+//You can put commands in groups
```
Comment (lhjt) [inline docs/guides/int_framework/samples/intro/groupmodule.cs]:
```suggestion
// You can put commands in groups
```

**Review by lhjt — CHANGES_REQUESTED** [review]
Please add a space after the `//` in your comments

File: docs/guides/int_framework/samples/intro/groupmodule.cs (hunk 2 above)
Comment (Misha-133) [reply docs/guides/int_framework/samples/intro/groupmodule.cs]: Fixed 🤔

**Review by lhjt — APPROVED** [review]

**Pull request merged**